JS1068_JS202_Getting_handled_onthe_DOM_challenge2/app.js

```diff
@@ -0,0 +1,7 @@
+// Complete the challenge by writing JavaScript below
+const listItems = document.getElementsByTagName('li');
+const colors = ["#C2272D", "#F8931F", "#FFFF01", "#009245", "#0193D9", "#0C04ED", "#612F90"];
+
+for(var i = 0; i < colors.length; i ++) {
```

Comment:
use `let` instead of `var`
```suggestion
for(let i = 0; i < colors.length; i ++) {
```